pages/checkOut.tsx

```diff
@@ -1,5 +1,34 @@
+import CheckInOutForm from '../components/CheckInOutForm'
+import { Box, Button } from '@mui/material'
+import { Typography } from '@mui/material'
+
 export default function CheckOutPage() {
   return (
-    <h1>Check Out</h1>
+    <Box
+      sx={{
```

Comment:
we wont ever really need to touch actual CSS thanks to MUI. to add padding/margin use the `p[t|b|l|r|x|y|'']` and the same for margin. for the card, use https://mui.com/material-ui/react-card/

Reply:
442b651 uses the MUI way of adding styling for most things and changes the card to use the MUI Card API. Let me know what you think!